## Abstract

Phenotypic traits evolve in a concerted manner due to developmental and functional interactions, mediated by the dynamics of adaptive landscapes; the codependence among traits arising from these three factors is captured by genetic ($\mathbf{G}$) and phenotypic ($\mathbf{P}$) covariance matrices.
Mammalian skull development produces an intricate pattern of tissue organization and mutual signaling that integrates this structure, although the set of functions it performs is quite disparate.
The interplay between these interactions, and their relationships with the adaptive landscape may thus influence divergence in covariance structure among sister lineages.
Here, we evaluate the stability of phenotypic covariance structure in skull size and shape along the diversification of Anthropoid Primates under a phylogenetic framework.
We estimate diversity in covariance structure, testing hypotheses concerning the phylogenetic distribution of $\mathbf{P}$-matrix variation; a second approach allows us to pinpoint which traits are associated with such variation.
Tests for the divergence in matrix structure indicate that most changes occurred in the basal split between Platyrrhini and Catarrhini; such changes occur within both Orbital and Basicranial trait sets, while Oral, Nasal and Vault trait sets present stable associations along the Anthropoid phylogeny.
Therefore, changes in $\mathbf{P}$-matrix structure among Anthropoids are restricted to trait sets whose functional significance is associated with the accomodation of the two precursor tissues that compose the skull, while the stability in the remaining regions hints at the stability of the underlying functional relationships imposed by the adaptive landscape.

\newpage

```{r options_ppca, include = FALSE}
opts_chunk $ set (fig.path = 'Figures/', dev = 'pdf')
```

```{r captions_ppca, include = FALSE}
captions $ fig.matrix_variation <- 'Representation of the steps used to characterize covariance matrix variation. In (a), the set of covariance matrices $\\mathbf{A}$, $\\mathbf{B}$ and $\\mathbf{C}$ in the neighbourhood of $\\mathbf{M}$ are projected into an Euclidean space and eigentensors are estimated (PM1 and PM2); in (b), these eigentensors are rotated to incorporate phylogenetic relatedness; in (c), covariance matrices at the upper and lower bounds of the confidence intervals for each axis are returned back to the original manifold. See text for more details. \\label{fig:matrix_variation}'

captions $ fig.eval <- 'Posterior distribution of eigenvalues obtained for pPCs. Positive eigenvalues are associated with phylogenetic signal; negative eigenvalues are associated with convergence. \\label{fig:eval}'

captions $ fig.loess <- 'Distribution of mean SRD scores for each trait along the entire ordered set of phylogenetic PCs. \\label{fig:loess}'

captions $ fig.moran <- 'Posterior bivariate distribution of variances explained by each phylogenetic PC *versus* Moran\'s Indexes estimated for each axis. \\label{fig:moran}'

captions $ fig.phylo_full <- 'Phylogenetic decomposition of $\\mathbf{P}$-matrix variation. (a) Decomposition of matrix diversity over the phylogenetic hypothesis for Anthropoidea; the size of each circle indicates the percentage of diversity on each node, according to the legend. (b) Mean $\\mathbf{P}$-matrices of each species projected over the first three and the last phylogenetic Principal Components (G1-3 and L1, respectively); cell colors represent projection values, according to the legend. \\label{fig:phylo_full}'

captions $ fig.srd_gl <- 'Covariance structure variation associated with the first three and last pPCs (Global 1-3 and Local 1, respectively), represented using posterior mean SRD scores. Dotted lines represent average SRD scores for each comparison. Traits are colored according to their assocation with each skull region, according to the legend. \\label{fig:srd_gl}'

captions $ fig.srd_shape <- 'Mean SRD scores for the first three and the last phylogenetic PCs. Global pPCs 1-3 and Local pPC 1, respectively, are represented over the mean skull shape of diana guenons (*Cercopithecus diana*). \\label{fig:srd_shape}'

### if thesis

captions $ fig.rs_riem <- 'The relationship between Riemannian Distances and Random Skewers among pairs of $\\mathbf{P}$-matrices. Each observation is the posterior mean value for each statistic, estimated from $100$ posterior samples obtained for each OTU. Colors indicate the lower sample size in each pair, according to the legend. \\label{fig:rs_riem}'

captions $ fig.ss_local1 <- 'Relationship between sample sizes and the first Local phylogenetic Principal Component. \\label{fig:ss_local1}'
    
```

# Introduction

Phenotypic traits evolve in a concerted manner either because of shared genetic and developmental processes or joint effects on fitness [@gould_spandrels_1979; @lande_quantitative_1979; @felsenstein_phylogenies_1988; @zeng_long-term_1988], and response to natural selection is maximized when both effects coincide, while discordance between them may hinder a direct response [@schluter_adaptive_1996].
The additive genetic covariances among traits ($\mathbf{G}$) and the partial regression coefficients of fitness on traits ($\beta$) provide linear approximations for both these effects in the characterization of phenotypic change across generations [@lande_quantitative_1979; @rice_general_2002].
Therefore, the additive covariances in $\mathbf{G}$ represent the codependency among traits due to pleiotropy and linkage disequilibrium, characterizing a linear genotype/phenotype map centered on the mean phenotype [@cheverud_developmental_1996; @wagner_homologues_1996; @wagner_eigenvalue_1984].

However, the structure of pleiotropic interactions depends on the local curvature of the genotype/phenotype map, traditionally represented in quantitative genetics as either dominance or epistasis [@rice_evolution_1998; @wolf_developmental_2001; @rice_developmental_2004].
The effect of epistatic *loci* on the modulation of pleiotropic interactions has been identified in experimental settings [e.g. @cheverud_pleiotropic_2004; @wolf_epistatic_2005; @pavlicev_genetic_2008], thus indicating that populations may harbor genetic variation in the linear components of genotype/phenotype maps.
Genetic covariances among phenotypic traits then evolve as a consequence of shifts in allele frequencies in these *loci*, for example in response to genetic drift and founder effects [@goodnight_quantitative_2000; @brito_evolution_2005].
local features of the adaptive landscape may also impact genetic covariances among traits, as $\mathbf{G}$ is thought to match the patterns imposed by stabilizing selection and mutational effects [@lande_genetic_1980; @cheverud_quantitative_1984; @jones_mutation_2007].
It is noteworthy that changes in genetic covariances due to the non-linear structure of adaptive landscapes can be explained just by considering shifts in linkage disequilibrium among *loci* [@turelli_phenotypic_1988], without any consideration to epistatic pleiotropy.
However, recent experimental data [@careau_evolution_2015] and simulation-based models [@jones_effects_2012; @melo_directional_2015] have demonstrated the effects of directional selection on the structure described by $\mathbf{G}$, thus indicating that the linear component of adaptive landscape can also have an impact on genetic covariances due to genetic variation in pleiotropic interactions.

The relationship between adaptive landscapes and intrapopulational covariance structure is mediated through performance [@arnold_morphology_1983], which may be thought as a dynamical property of phenotypes; thus, the separation between developmental and functional interactions as two distinct factors shaping phenotypic covariance structure is blurry at least [@cheverud_developmental_1996; @zelditch_epigenetic_2011].
For example, in the mammalian skull, precursor tissues (paraxial mesoderm and neural crest) originate two distinct regions, Face and Neurocranium, which exhibit marked contrasts in terms of functions which they perform, interactions with soft tissues, and response to developmental milestones, such as birth or weaning [@hallgrimsson_mouse_2008; @lieberman_epigenetic_2011].
Thus, the Oral, Nasal, and Zygomatic regions are associated with the Face and are responsible for mastication, respiration, and the attachment of the muscular apparatus involved in mandibular articulation.
The Vault, Orbit and Base regions are associated with the Neurocranium and are responsible for encasing and protecting both brain and eye, and for supporting and connecting the skull with the remaining organismal systems; the Vault also houses muscle attachment sites associated with mandibular articulation, indicating that, to some degree, regions may be involved in more than one function and that some functions may be shared between them.
The contrast between these regions is the result of distinct gene expression profiles, which are further changed by the diffusion of signaling factors, thus generating a feedback loop of cell and tissue differentiation [@turing_chemical_1952; @marcucio_molecular_2005; @meinhardt_models_2008; @franz-odendaal_epigenetics_2011; @xu_correlations_2015].

These signaling factors may target specific cell lineages, but the contact between adjoining tissues may produce correlated changes between them due to mechanical interactions and through mutual signaling cascades that induce specific behaviors in neighbor tissues, and such interactions are necessary for the proper development of both tissues [@cheverud_effects_1992; @ravosa_masticatory_2000; @jiang_tissue_2002; @marcucio_mechanisms_2011].
For instance, cranial Vault growth, which occurs on the final stages of pre-natal development, is a result of the tension exerted by the growing brain on its encircling membranes, inducing them to secrete signaling factors which promote bone growth [@opperman_cranial_2000; @rice_molecular_2003], and such mechanism promotes a tight association between the Vault and the brain. 
Furthermore, post-natal facial growth is induced by muscular activity related to masticatory function, and this effect dominates post-natal skull growth [@zelditch_ontogeny_1992; @herring_muscle-bone_2011], since muscle activity mainly affects the focal Oral region in which muscular forces are exerted, but other skull regions to which these muscles are attached are also affected to a lesser extent, such as the Zygomatic and Vault.
Thus, development is composed of a series of such events, and it may be difficult to isolate the effect each individual process has on covariance structure, given their spatial overlap [@hallgrimsson_deciphering_2009]; hence, both the temporal hierarchy and the spatial organization of developmental and functional interactions influence the patterns embedded in genetic covariance structure.
Moreover, these molecular aspects of developmental regulation motivate distinct definitions for pleiotropy [@zelditch_epigenetic_2011]: one in which the focal *locus* affects a series of developmental processes within a single cell lineage (*intrinsic, genetic effect*), and another in which processes in one cell lineage depends on the expression of the focal *locus* on another (*extrinsic, epigenetic effect*); thus, such definitions impart spatial structure to pleiotropic and epistatic interactions.

Empirical evidence on long-term changes on genetic covariance structure [comparative quantitative genetics; @steppan_comparative_2002] rely on the correspondence between  $\mathbf{G}$ and its phenotypic counterpart $\mathbf{P}$, since estimating $\mathbf{G}$ demands high sample sizes and nonetheless such estimate is prone to substantial error, since sample units are families rather than individuals [@meyer_estimating_1991; @houle_estimating_2015].
The similarity in covariance structure between $\mathbf{P}$ and $\mathbf{G}$ ['Cheverud's Conjecture'; @cheverud_comparison_1988; @roff_estimation_1995], has been supported in different trait systems and organisms [e.g.: @waitt_genetic_1998; @dochtermann_testing_2011; @garcia_quantitative_2014], indicating that environmental effects ($\mathbf{E}$) are either uncorrelated or exhibit patterns similar to $\mathbf{G}$, since these effects exert their influence over phenotypes through the same developmental processes in which genetic variation is structured [@rice_general_2002; @rice_developmental_2004].

The comparative analysis of phenotypic covariance structure shows that $\mathbf{P}$ can remain stable in macroevolutionary scales [e.g. @marroig_comparison_2001; @oliveira_covariance_2009; @kolbe_convergent_2011], and such stability may be a consequence of the alignment between phenotypic covariance structure and the local features of the adaptive landscape acting over different lineages.
In New World Monkeys (Platyrrhini), divergence in body size among lineages is closely associated with shifts in diet composition [@rosenberger_evolution_1992].
Size represents the main feature of both genetic [@cheverud_morphological_1995; @cheverud_quantitative_1996] and phenotypic [@marroig_size_2005] covariance structure, and for some groups within New World Monkeys, such as Atelids and Callithrichines, divergence in body size is a direct consequence of directional selection [@marroig_size_2010; @marroig_modularity_2012].
On the other hand, dietary or locomotory divergence between sister lineages can rearrange the patterns expressed in $\mathbf{P}$ in some situations [@young_serial_2005; @young_development_2010; @monteiro_adaptive_2010; @haber_evolution_2015], and such reorganization hints at changes in the underlying architecture of the genotype/phenotype map, indicating that the patterns embedded in the adaptive landscape can overcome constraints imposed by development [@jamniczky_comparison_2009].

The adaptive radiation of phylostomid bats, for instance, has involved rearrangements of mandibular phenotypic covariance structure [@monteiro_adaptive_2010], as the dietary divergence in this group from a probable insectivore ancestor towards more specialized diets, such as frugivory or sanguivory, imply distinct functional relationships among mandibular components; thus, such reorganization may indicate heterogeneity in the non-linear aspects of the adaptive landscape for different phylostomid lineages.
For bats in general, their specialized locomotory behavior is associated with the decoupling between fore- and hindlimb covariance structure when compared to other mammalian lineages [@young_serial_2005]; to a lesser extent, a similar pattern can be observed in hominoids when compared to remaining Anthropoids [@young_development_2010].
Another example of divergence in covariance structure among lineages due to changes in functional demands are those observed in ruminants [@haber_evolution_2015], which may related to increased metabolic requirements for occupying open habitats, as the divergence between Bovidae and Cervidae is associated with increased Nasal integration in the former.
Thus, divergence among sister lineages may impart a reorganization of both genetic and phenotypic covariance structure of phenotypic traits if such divergence is associated with distinct functional relationships, represented by the non-linear components of adaptive landscape.

Currently, there is an abundance of methods dedicated to the comparison of covariance structures [e.g. @krzanowsky_between-groups_1979; @phillips_hierarchical_1999; @cheverud_comparing_2007], and such methods are often focused on constructing hypothesis of similarity or dissimilarity; however, in some situations, either hypotheses can be rejected in the same comparison [@ackermann_phenotypic_2000; @marroig_comparison_2001; @haber_evolution_2015].
By expressing covariance matrix (dis)similarities with a single metric, these methods lack an explicit way of describing the differences in covariance structure between pairs of covariance matrices; furthermore, these methods also lack a direct manner to incorporate phylogenetic relatedness in pairwise comparisons.
While extensions for these methods have been proposed to deal with the first limitation [e.g. @hansen_measuring_2008; @hine_characterizing_2009; @marroig_selection_2011], the second issue is usually resolved by comparing the set of pairwise matrix comparisons with the set of phylogenetic distances among lineages [e.g. @marroig_comparison_2001; @oliveira_covariance_2009].
Although such comparison provides a first approximation to this problem, it has the same problems as comparing covariance matrices themselves, that is, summarising the relationship between patterns expressed in $\mathbf{P}$ or $\mathbf{G}$ in a set of lineages and their phylogenetic relatedness to a single value.

### Objectives

The interplay between developmental and functional interactions, and their relationships with the topology of adaptive landscapes may influence the divergence between covariance structure in sister lineages. In the present work, we evaluate the stability of phenotypic covariance structure in skull size and shape along the diversification of Anthropoid Primates.
We build upon approaches proposed by other authors [@marroig_selection_2011; @aguirre_comparing_2013; @haber_evolution_2015] in order to explicitly incorporate phylogenetic relationships into the comparative analysis of covariance structure, under the hypothesis that different cranial regions will exhibit different degrees of stability among sister lineages, thus producing a non-random pattern of changes in covariance structure.

# Methods

## Sample

Our sample consists of `r sum (Aux $ sample.size)` individuals in `r length (Aux $ sample.size)` species, distributed throughout all major Anthropoid clades above the genus level, comprising all Platyrrhini genera and a substantial portion of Catarrhini genera.
We associate this database with a ultrametric phylogenetic hypothesis for Anthropoidea (\autoref{fig:phylo_model}), derived from Springer *et al.* [-@springer_macroevolutionary_2012].

Individuals in our sample are represented by 36 landmarks; these landmarks were registered using a Polhemus 3Draw and a Microscribe 3DS for Platyrrhini and Catarrhini, respectively.
Twenty-two unique landmarks represent each individual (Figure \ref{fig:landmarks}), since fourteen of the 36 registered landmarks are bilaterally symmetrical. For more details on landmark registration, see @marroig_comparison_2001 and @oliveira_covariance_2009.
Databases from both previous studies were merged into a single database, retaining only those individuals in which all landmarks from both sides were present.
In the present work, we considered only covariance structure for the symmetrical component of variation; therefore, prior to any analysis, we controlled the effects of variation in assymmetry.
We followed the procedure outlined in @klingenberg_shape_2002 for bilateral structures by obtaining for each individual a symmetrical landmark configuration, averaging each actual shape with its reflection along the sagittal plane.

We used this database to obtain local shape variables [@marquez_measurement_2012], which represent infinitesimal volumetric expansions or retractions, calculated as the natural logarithm determinants of derivatives of the TPS function between each individual in our sample and a reference shape (in our case, the mean shape for the entire sample, estimated from a Generalized Procrustes algorithm).
Such derivatives were evaluated at the midpoints between pairs of landmarks represented in \autoref{fig:landmarks}, for a total of 38 local shape variables.

After obtaining these values, we estimated covariance $\mathbf{P}$-matrices for size (represented by the natural logarithm of Centroid Size) and local shape variables after removing fixed effects of little interest in the present context, such as sexual dimorphism, for example.
These effects were removed through a multivariate linear model adjusted for each species, according to \autoref{fig:phylo_model}.
We adjusted such models under a Bayesian framework, sampling $100$ residual covariance matrices from the posterior distribution of each model.
These distributions allow us to estimate uncertainty for any parameters derived from these matrices as credibility intervals; furthermore, since posterior distributions are conditional upon the prior distribution we used --- a uniform Wishart distribution --- every matrix sampled from these posterior distributions is also a realization of a Wishart distribution, therefore positive-definite regardless of sample size [@gelman_bayesian_2004]; in this framework, lower sample sizes imply in broader and less informative credibility intervals.
For each posterior sample, we estimated geometric mean covariance matrices, since such mean respects the underlying structure of the Riemannian manifold in which positive-definite symmetric matrices lie [@moakher_differential_2005; @moakher_averaging_2006]; thus, these mean $\mathbf{P}$-matrices are also positive-definite, regardless of sample size.
For each species, we ran independent models, with 13000 iterations of MCMC sampling, discarding the 3000 initial iterations as a burnin period and further sampling one covariance matrix per 100 iterations to avoid autocorrelations induced by sequential sampling.

## Phylogenetic Decompostion of Matrix Diversity

In order to evaluate the distribution of covariance structure diversity during Anthropoid diversification, we estimated Riemannian distances among all pairs of mean $\mathbf{P}$-matrices, according to the definition given by @mitteroecker_ontogenetic_2009; for any pair of positive-definite covariance matrices $\mathbf{C}_i$ and $\mathbf{C}_j$ of size $p \times p$, the distance $d(\mathbf{C}_i, \mathbf{C}_j)$ is given by
\begin{equation}
d(\mathbf{C}_i, \mathbf{C}_j) = \sqrt{\sum_{k = 1}^p \ln^2 \lambda_k(\mathbf{C}_i\mathbf{C}_j^{-1})}
\label{eq:riemdist}
\end{equation}
where $\lambda_k(\cdot)$ refers to the $k$-th eigenvalue obtained from the spectral decomposition of a given matrix, in this case the product $\mathbf{C}_i\mathbf{C}_j^{-1}$.
This distance among pairs of $\mathbf{P}$-matrices is negatively correlated with Random Skewers comparisons (\autoref{fig:rs_riem}), a measurement of matrix similarity explored elsewhere [@cheverud_comparing_2007]; such correlation indicates that our exploration of Riemmanian distances among $\mathbf{P}$-matrices could be extended to other types of measurement of similarity or dissimilarity among these objects without hindrances. 

```{r rs_riem, echo = FALSE, fig.width = 8, fig.height = 8, fig.cap = captions $ fig.rs_riem}
if (thesis) ppca.extra $ rs.riem
```

Using these distances among $\mathbf{P}$-matrices, we estimate matrix diversity at each node of the phylogenetic tree of Anthropoidea using a measurement of the weighted distance among the distributions of matrix distances for the two descending edges, based on @pavoine_decomposition_2010.
For a fully resolved tree, diversity $w_i$ on node $i$ is given by
\begin{equation}
w_i = \frac{1}{2} \frac{n_\alpha n_\beta} {n_i n_T} D_{\Delta}^2(P_\alpha, P_\beta)
\label{eq:div}
\end{equation}
where $\alpha$ and $\beta$ represent the subsets of descendants from node $i$, and $n$ refers to the number of OTUs on each set ($n_i$ for the total descendants of node $i$; $n_T$ for the total number of OTUs considered; $n_\alpha$ and $n_\beta$ for the size of descending subsets).
$D_{\Delta}(P_\alpha, P_\beta)$ represents the actual distance between the two  distributions $P_\alpha$ and $P_\beta$ for descending nodes, as formulated by @rao_diversity_1982:
\begin{equation}
D_{\Delta} (P_\alpha, P_\beta) =
\sqrt{2 \bigg( 2 H_{\Delta} \bigg( \frac{P_\alpha + P_\beta}{2} \bigg) -
H_{\Delta}(P_\alpha) - H_{\Delta}(P_\beta) \bigg)}
\label{eq:distdist}
\end{equation}
where
\begin{equation}
H_{\Delta} (P) = \sum_{i,j \in P} \frac{d^2(\mathbf{C}_i, \mathbf{C}_j)}{2}
\label{eq:rao}
\end{equation}
represents Rao's quadratic entropy among Riemannian distances $d(\mathbf{C}_i, \mathbf{C}_j)$ as defined in \autoref{eq:riemdist}.

Following the framework estabilished by @pavoine_decomposition_2010, diversity $w_i$ can be normalized as $v_i = w_i / \sum_i w_i$ to represent the percentage of diversity with respect to the total diversity on the phylogenetic tree.
We test three different hypothesis regarding the distribution of $v_i$ values through Anthropoid diversification: (1) that $\mathbf{P}$-matrix diversity is concentrated in a single node; (2) that $\mathbf{P}$-matrix diversity is concentrated in a reduced number of nodes; (3) that $\mathbf{P}$-matrix diversity is skewed towards either the root or tips of the phylogeny, in a two-tailed test.
We test each hypothesis against the null hypothesis that the distribution of matrix diversity is randomly arranged over the phylogeny; such null hypothesis is represented by randomizing the association between terminal branches and covariance matrices, constructing $9999$ distributions of $v_i$ values that represent this scenario.
Each test is carried out using a different parameter derived from the distribution of $v_i$ values [described in detail by @pavoine_decomposition_2010], comparing the actual value obtained with a null distribution constructed using permutations.
The third hypothesis can be tested either by considering only the topology of the tree and by also considering branch lengths; both tests are similar to Blomberg's [-@blomberg_testing_2003] $K$ test, as they search for a phylogenetic signal in covariance structure diversity.

## Characterizing Covariance Matrix Variation

The tests described in the previous section allow us to pinpoint which nodes contribute mostly to divergence in covariance structure; however, these tests are not designed to properly describe the actual changes in $\mathbf{P}$-matrix structure that are responsible for such divergence. To actually represent these changes in a comprehensible manner, we combine a number of ordination techniques to reduce the dimensionality of the manifold that contains covariance matrices of size $p \times p$ (\autoref{fig:matrix_variation}).

For a Riemannian manifold, there exists at least one bijective function defined in the neighbourhood of a given covariance matrix $\mathbf{M}$ that maps the manifold to an Euclidean space --- a hyperplane with $p (p - 1) / 2$ dimensions also contained in $\mathbb{R}^{p \times p}$ --- and equips the manifold with a notion of inner product, thus allowing the construction of an orthonormal basis that can be used to describe variation in $\mathbf{P}$-matrix structure.
For a covariance matrix $\mathbf{X}$ in the neighbourhood of $\mathbf{M}$,
\begin{equation}
f(\mathbf{X}) = \log (\mathbf{M}^{- \frac{1}{2}} \mathbf{X} \mathbf{M}^{- \frac{1}{2}})
\label{eq:map}
\end{equation}
represents one possible function.
Here, the logarithm operator refers to matrix logarithm; for symmetric positive-definite matrices, this transformation is equivalent to applying the usual logarithm function to the eigenvalues of such matrix and reverting the spectral decomposition.
The function defined in \autoref{eq:map} also transforms the Riemannian distance among covariance matrices defined in \autoref{eq:riemdist} into Euclidean distances between transformed matrices [@moakher_differential_2005].

![`r captions $ fig.matrix_variation`](Figures/matrix_variation.pdf)

We defined the average matrix among all sampled $\mathbf{P}$-matrices as the location parameter $\mathbf{M}$ to map the entire set of posterior $\mathbf{P}$-matrices into an Euclidean space.
We then used these $\mathbf{P}$-matrices to produce axes of matrix variation using an eigentensor decomposition [@basser_spectral_2007; @hine_characterizing_2009] obtaining a set of eigentensors and eigenvalues that summarise matrix variation (\autoref{fig:matrix_variation}a).
As a consequence of using the mean covariance matrix for the entire sample over \autoref{eq:map}, the projections over eigentensors we obtained are naturally centered on $\mathbf{M}$. 

We used the projections of $\mathbf{P}$-matrices over these eigentensors as traits in a phylogenetic Principal Component Analysis [pPCA; @jombart_putting_2010], which produces a new set of axes of matrix variation that considers both trait dispersal and phylogenetic relationships among OTUs simultaneously. If $\mathbf{Z}$ represents a matrix with projections of $n$ $\mathbf{P}$-matrices over each eigentensor on its columns, phylogenetic PCs are the eigenvectors obtained from
\begin{equation}
\frac{1}{2n} \mathbf{Z}^t(\mathbf{W} + \mathbf{W}^t) \mathbf{Z}
\end{equation}
where $\mathbf{W}$ represents the matrix of phylogenetic distances between OTUs; here, the distance $w_{ij}$ between tips $i$ and $j$ is the sum of branch lengths from their last common ancestor to both tips. Other definition of phylogenetic distances may be used [see @jombart_putting_2010]; the results we show here were not changed by considering different measures of phylogenetic distance, regardless of whether these distances consider branch lengths among OTUs or not.

Such analysis produces both positive and negative eigenvalues, which are respectively associated with variation close to the root of the tree ('Global') and variation close to the tips ('Local') in matrix structure (pPC1 and pPC2 in \autoref{fig:matrix_variation}b, respectively). @pavoine_decomposition_2010 argues that this contrast between Global and Local components in phylogenetic PCs reflects phylogenetic signal and convergence in trait values, respectively, as observed in the distribution of Moran's [-@moran_interpretation_1948; -@moran_notes_1950] autocorrelation Indexes for each axis constructed in this manner. This index can be understood as the degree onto which an observed value in a given OTU is determined by the values on its phylogenetic "neighborhood" (as expressed by $\mathbf{W}$), in a similar manner to autoregressive models [@cheverud_autocorrelation_1985; @cheverud_quantitative_1985].

For each pPC obtained in this manner, we obtained two covariance matrices by estimating the upper and lower limits of the 95\% confidence interval for each axis and mapping these values back to the manifold of symmetric positive-definite matrices (\autoref{fig:matrix_variation}c), defining the inverse operation associated with \autoref{eq:map} as
\begin{equation}
f^{-1}(\mathbf{X}) = \mathbf{M}^{\frac{1}{2}}\exp(\mathbf{X})\mathbf{M}^{\frac{1}{2}}
\label{eq:inv}
\end{equation}
where the exponential operator refers to matrix exponential. We used these covariance matrices to describe matrix variation associated with each axis comparing each pair of matrices with the Selection Response Decomposition tool [@marroig_selection_2011], in order to pinpoint which traits are associated with the divergence in covariance structure associated with each pPC.

In order to characterize such divergence in covariance structure with respect to the uncertainty in $\mathbf{P}$-matrix estimation, we carried out the analyses described in this section with both mean $\mathbf{P}$-matrices obtained from posterior samples, and with posterior samples themselves, obtaining $100$ sets of phylogenetic PCs and $100$ sets of paired covariance matrices for each axis, thus allowing us to estimate a posterior distribution of mean SRD scores for each trait in all pPCs. We use the phylogenetic PCA estimated over mean $\mathbf{P}$-matrices in order to represent the phylogenetic patterns described by each pPC.

We use the posterior distribution of mean SRD scores over traits and pPC to investigate whether these changes in trait-specific covariance structure along Anthropoid diversification are randomly distributed with respect to the skull regions delimited in \autoref{tab:dist} by comparing SRD scores estimated within each region for all pPCs.
The association between cranial traits and such regions reflect their functional significance and developmental origins.

### Software

We performed all analyses under R `r paste (version $ major, version $ minor, sep = '.')` [@r_core_team_r:_2015].
We fitted Bayesian linear models for estimating posterior $\mathbf{P}$-matrix samples using the *MCMCglmm* package [@hadfield_mcmc_2010].
Source code for the eigentensor decomposition can be found at <http://github.com/wgar84>.
The phylogenetic decomposition of diversity was provided by @pavoine_decomposition_2010 in their Supplemental Material, while pPCA is implemented in the *adephylo* package [@jombart_adephylo:_2010]; the SRD method is provided by the *evolqg* package [@melo_evolqg_2015].
In order to obtain symmetrical landmarks configurations, we used code provided by Annat Haber, available at <http://life.bio.sunysb.edu/morph/soft-R.html>. 

# Results

The distribution of covariance matrix diversity along the Anthropoid phylogeny (\autoref{fig:phylo_full}a) indicates that the divergence between Catarrhini and Platyrrhini contributes to approximately 10\% of all covariance matrix diversity; within New World Monkeys, the divergence between Atelidae and Cebidae contributes with 4\% to covariance structure diversity, while for Catarrhines, the divergence between Hominoidea and Cercopithecoidea contributes 3\% to overall covariance matrix diversity. The remaining $\mathbf{P}$-matrix diversity is distributed along the tree, with a consistent decay of explained diversity the closer any given node is from terminal branches.
With respect to the hierarchy of tests exploring the phylogenetic distribution of matrix diversity (\autoref{tab:riem_decdiv}), all tests reject their null hypotheses of random arrangements of diversity along the Anthropoid phylogeny, thus indicating that covariance matrix diversity exhibits some degree of phylogenetic structure, with these few basal nodes contributing to a greater extent to such diversity.

\input{Tables/riem_decdiv} <!-- tab:decdiv -->

```{r phylo_full, echo = FALSE, fig.width = 10, fig.height = 12, fig.cap = captions $ fig.phylo_full, warning = FALSE}
Tree.Plots $ ppca
```

The eigenvalue distribution of phylogenetic Principal Components that describe $\mathbf{P}$-matrix structure (\autoref{fig:eval}) shows that individual Global components surpass their Local counterparts in terms of explained variance, such that the first Global component has a larger contribution than the first Local component to interspecific $\mathbf{P}$-matrix variation. While there are less positive than negative eigenvalues, Global components explain more than half (57\%) of the total $\mathbf{P}$-matrix variation, considering mean absolute eigenvalues obtained from their posterior distribution. The posterior distribution of Moran's Index for phylogenetic Principal Components (\autoref{fig:moran}) is assymetric towards positive values, also indicating that the similarity produced by phylogenetic conservatism is greater than the similarity produced by convergence in $\mathbf{P}$-matrix structure.

```{r eval, fig.width = 6, fig.height = 6, echo = FALSE, warning = FALSE, message = FALSE, fig.cap = captions $ fig.eval}
post.ppca $ plot.ppca.eval + xlab ('Phylogenetic Principal Component') +
  scale_x_continuous(breaks = c(1, 31, 61, 91))
```

```{r moran, fig.width = 6, fig.height = 6, echo = FALSE, warning = FALSE, message = FALSE, fig.cap = captions $ fig.moran}
post.ppca $ plot.var.moran
```

Considering the distribution of matrix projections for each OTU on these pPCs (\autoref{fig:phylo_full}b), we observe that the first Global pPC separates New World and Old Monkeys, while the second Global pPC consists of a contrast between Atelids and Cebids, and the third Global pPC generally contrasts Hominids with the remaining Anthropoids, thus indicating a pattern of $\mathbf{P}$-matrix variation consistent with those observed in the diversity decomposition summarized in \autoref{fig:phylo_full}a. The first Local pPC consists of localized contrasts between sister species, such as the two representatives of *Saimiri*, for instance; however, matrix projections over this axis can be explained by the effect of sample sizes (\autoref{fig:ss_local1}). These localized contrasts can thus be explained on the account of substantial sample size differences between sister species.

```{r srd_gl, echo = FALSE, fig.width = 10, fig.height = 10, warning = FALSE, message = FALSE, fig.cap = captions $ fig.srd_gl}
post.ppca $ plot.srd.gl12 +
  scale_color_brewer(name = 'Region',
                     labels = c('Size', 'Oral', 'Oral/Nasal', 'Nasal', 'Zygomatic',
                       'Orbit', 'Base', 'Vault'),
                     palette = 'Dark2')
```

```{r ss_local1, eval = thesis, echo = FALSE, fig.width = 8, fig.height = 8, fig.cap = captions $ fig.ss_local1}
ppca.extra $ ss.local1
```

The variation in trait-specific covariance structure described by these four phylogenetic Principal Components, as captured by comparing the posterior distribution of confidence intervals for each axis using Selection Response Decomposition (\autoref{fig:srd_gl}) indicates that, regardless of which axis is considered, traits with lower posterior SRD scores are usually localized in either Orbit or Basicranium, along with log Centroid Size, which represents the covariance structure associated with allometric relationships.
Traits in remaining skull regions (Oral, Nasal, Zygomatic, Vault) consistently exhibit higher SRD scores, thus indicating a more stable covariance structure associated with these regions throughout Anthropoid diversification. 

```{r loess, fig.width = 10, fig.height = 10, echo = FALSE, warning = FALSE, message = FALSE, fig.cap = captions $ fig.loess}
alt.spline
```

The overall distribution of average posterior SRD scores along the entire set of phylogenetic PCs (\autoref{fig:loess}) indicates that this behavior detailed on \autoref{fig:srd_gl} for G1-3 and L1 is the norm also for the remaining pPCs, that is, Orbit and Basicranial traits along with allometric relationships consistenly have lower average SRD scores than other skull regions. Notice that the explained amount of variance for intermediate pPCs (roughly from pPC 31 to 61) is very close to zero (\autoref{fig:eval}); thus, although matrices describing confidence intervals in these axes can be recovered, the pattern expressed by then should not be taken into account. 

# Discussion

Since its conception, the hypothesis that functional interactions among morphological traits shape their phenotypic covariance structure [@olson_morphological_1958] has been complemented with the notion that developmental interactions mediate the functional relationships among traits dynamically; thus, it is difficult to separate the relative contribution of either development or function to phenotypic integration [@cheverud_developmental_1996], especially considering that the structure of developmental interactions is thought to match the pattern of optimal functional interactions [@lande_genetic_1980; @cheverud_quantitative_1984; @jones_mutation_2007; @wagner_homologues_1996], which further entangles both phenomena.
Thus, changes in covariance structure between sister lineages should be associated with the interplay between functional and developmental interactions.

Our results indicate that the changes in phenotypic covariance structure associated with all phylogenetic principal components follow a similar pattern with respect to changes in trait-specific covariances (Figures \ref{fig:srd_gl} and \ref{fig:loess}).
Such changes are mostly associated with either Basicranial and Orbital trait sets, along with allometric relationships, represented by the covariances between log Centroid Size and local shape variables.
It is noteworthy that while the comparison between covariance matrices in the limits of each phylogenetic PC represents the overall pattern of dissimilarity in trait-specific covariance structure associated with each axis, the actual values obtained represent only a fraction of the overall divergence between Anthropoid lineages.
Given any two $\mathbf{P}$-matrices associated with a pair of species, the actual SRD comparison between then will be some linear combination of their divergences along each pPC axis.

Orbital traits are mostly associated with the development of the postorbital bar in Euprimates, as opposed to Plesiadapiforms, and in Anthropoids it fully develops into the orbital cavity and postorbital wall [@ravosa_euprimate_2004].
The origins of such structure have been linked to the distribution of masticatory loadings around the comparatively large primate eye for the maintenance of a stable, forward-facing visual field even during feeding behavior [@ravosa_masticatory_2000].
Although most of the cranial Vault originates from intramembranous ossification induced by the growing brain and thus derived from the neural crest, the influence of mesoderm-derived condensations and its pattern of endochondral ossification is necessary for the proper development of the fronto-nasal and fronto-zygomatic sutures, which affect the brow ridge and both medial and lateral orbital walls [@jiang_tissue_2002].

In the same manner, the Basicranium originates from a set of thirteen condensations derived from both precursors, which exhibit a mosaic pattern of both endochondral and intramembranous ossification [@lieberman_epigenetic_2011].
Such processes occurs early during skull development, and the spatial overlap of developmental processes [@hallgrimsson_deciphering_2009] may also explain Basicranium variation in covariance structure.
Moreover, the angulation between anterior and posterior elements of the Basicranium has significantly changed during primate evolution, and such property appears to have evolved in coordination with Facial growth relative to the cranial Vault, accomodating both structures on each other [@scott_cranial_1958; @lieberman_primate_2000; @lieberman_spatial_2008]. 
Therefore, both Orbit and Basicranium are located in the boundary between domains of two precursor tissues that originate osteological elements in the skull, and their proper development is affected by both of them; furthermore, their functional significance is also associated with the accomodation of remaining structures.
These characteristic properties of their development may thus be sufficient to explain their divergence among lineages in terms of covariance structure.

On the other hand, the structure of phenotypic covariances for Oral, Nasal and Vault traits remained stable through the course of Anthropoid diversification. These regions exhibit more consistent patterns of developmental processes, when compared to the Orbit or Basicranium.
Both Oral and Nasal regions originate from mesoderm-derived tissue, and exhibit a great degree of interactions with soft tissue during prenatal development; the Oral region further suffers the influence of muscleskeletal interactions associated with Facial growth in postnatal development, which also contributes to its pattern of stability in covariance structure [@zelditch_ontogeny_1992; @herring_muscle-bone_2011; @lieberman_epigenetic_2011].
The cranial Vault originates mostly from precursors derived from the neural crest and exhibits a more regular pattern of growth, induced by the underlying brain [@lieberman_epigenetic_2011; @esteve-altava_beyond_2014].
While there are prenatal developmental processes associated with the integration of both structures [@marcucio_molecular_2005; @marcucio_mechanisms_2011] and postnatal muscle-bone interactions may be understood as a overall integrating factor, since the skull as whole is affected by such interactions, each of these regions is located within the bounds of precursor components, thus exhibiting a stable association between developmental processes that originate these regions and their functional aspects.

Finally, allometric relationship exhibit changes in covariance structure across phylogenetic principal components with magnitudes equivalent to Basicranial traits (Figures \ref{fig:srd_gl} and \ref{fig:loess}). While such relationships are explored elsewhere (`r if (thesis) 'Chapter 3' else 'Garcia *et al.*, in prep'`), it is noteworthy that these changes may be associated with the role of directional selection for body size, which shaped the extant diversity of New World [@marroig_size_2005; @marroig_size_2010] and Old World Monkeys [@cardini_geometric_2007; @cardini_variation_2008], as well as the more complex relationships between size and shape observed within Hominidae [@ackermann_detecting_2004; @mitteroecker_comparison_2004; @mitteroecker_evolutionary_2008; @schroeder_characterizing_2014].
To some extent, selection for body size produces mean shape differences among lineages in a manner consistent with the ancestral allometric relationships [@lande_quantitative_1979; @schluter_adaptive_1996], but selective pressures for size may alter such relationships depending on the structure of interactions between size, shape and ontogeny [@pelabon_relationship_2013; @pelabon_evolution_2014].

The phylogenetic distribution of changes in covariance strucuture (\autoref{fig:phylo_full}) reveals an even distribution of such changes throughout Anthropoid diversification, with three different instances in which more substantial changes have occurred, in order of the estimated $\mathbf{P}$-matrix diversity at each point (\autoref{fig:phylo_full}a): the divergence between New World and Old World Monkeys, between Atelidae and Cebidae, and between Hominoidea and the remaining Anthropoids.
Results from the phylogenetic principal component analysis (\autoref{fig:phylo_full}b) recover a similar pattern, considering the contribution of these lineages to each axis of matrix variation, and the posterior distributions of the associated eigenvalues for the first, second and third phylogenetic principal components (\autoref{fig:eval}) indicate that these cladogenetic events can be set apart in terms of their relative contributions to $\mathbf{P}$-matrix diversity.

Although the phenotypic covariance structure for Hominids has been regarded as the most disparate when compared to remaining Catarrhines [@oliveira_covariance_2009] and to mammals in general [@porto_evolution_2009], these findings may differ qualitatively from our results due to the use of traditional morphometric measurements, as opposed to the geometric morphometrics approach we use here.
Without any means of controlling the contribution of size variation, interlandmark distances will overestimate the influence of this source of variation over subtle shape relationships; thus, as the divergence in Hominid covariance structure is mostly related to differences in allometric relationships in this lineage (`r if (thesis) '\\autoref{fig:cac_homi}' else 'Garcia *et al.*, in prep'`), interlandmark distances will put more emphasis on these differences.
On the other hand, the actual pattern of divergence in the third phylogenetic component (\autoref{fig:phylo_full}b) contrasts the Hominoid lineage apart not only from its sister group, Cercopithecoidea, but also from remaining lineages.
Thus, the apparent greater contribution from the divergence between Atelids and Cebids to $\mathbf{P}$-matrix diversity may be a consequence of the phylogenetic structure imposed by either analyses, as the constrast produced by the third pPC is not a proper constrast between sister lineages, but rather between Hominoidea and the paraphyletic grouping obtained from removing this group.

However, both events are overshadowed by the separation between New World and Old World Monkeys, since this divergence is associated with an unequivocally higher eigenvalue, as indicated by their posterior distribution (\autoref{fig:eval}).
The subtle differences in $\mathbf{P}$-matrix diversity such separation represents may indicate that changes in the genetic architecture of cranial traits due to founder effects [@goodnight_quantitative_2000] may be a sufficient explanation for this divergence, considering that the effective population sizes in the ancestral population for all Platyrrhines probably plummeted during the transition between Africa and South America.
Such transition probably occured through the colonization of successive island environments that may have occurred in the South Atlantic Ocean during the Eocene [@oliveira_paleogeography_2009].

It is noteworthy that the pattern recovered by the first Local component (lower right panel of \autoref{fig:srd_gl}) also describes a pattern similar to that of Global components, considering that the distribution of lineages in this component scales with sample sizes on a logarithmic scale (\autoref{fig:ss_local1}).
Basicranial and Orbital traits exhibit overall lower covariances, both among themselves and between other regions, which contributes to more uncertainty in their estimation, and to a stronger sampling effect over their values.
However, it is also important to consider that these directions depicted as changes in trait-specific covariance structure using the SRD method are orthogonal in the Euclidean image of the exponential-logarithm map, and as such, they represent different aspects of covariance divergence among lineages, which here are concentrated in the same traits regardless of the phylogenetic structure considered.
The association between the first Local component with sample sizes hinders its interpretation as biologically meaningful, in the terms of Jombart's [-@jombart_putting_2010] notion that Local components should be related to convergence among phylogenetically disparate lineages; rather, this component seems to be associated with divergence in the statistical properties of estimated covariance matrices.

Regardless of such considerations with respect Local components, the prevalence of variation associated with positive, Global components in relation to variation in negative, Local components (Figures \ref{fig:eval} and \ref{fig:moran}) implies that the contribution of historical constraints is greater than the contribution of convergence among disparate lineages.
Considering that either process is associated with a stable structure of changes in covariance structure (Figures \ref{fig:srd_gl} and \ref{fig:loess}) related to some of the properties of skull development, these results stress out the overall stability of covariance structure in Anthropoids.
<!--- move along, nothing else to say here -->

### Conclusions

The stability of shape covariance structure in Anthropoids may be a consequence of either constraints on mammalian skull development or the prevalence of a constant pattern of functional relationships imposed by stabilizing selection; such dichotomy has already been pointed out by other authors [e.g. @marroig_comparison_2001; @porto_evolution_2009; @oliveira_covariance_2009].
In the present work, we tend to favor the second point of view, in light of the evidence for the evolution of genetic or phenotypic covariances under directional selection [@jones_effects_2012; @melo_directional_2015; @careau_evolution_2015] or under relaxation of stabilizing selection [@jamniczky_comparison_2009].
Furthermore, some available comparative data [@monteiro_adaptive_2010; @haber_evolution_2015] indicate that some mammal lineages, such as phylostomid bats and ruminants, have diverged in phenotypic covariance structure probably due to an heterogeneity in the shape of adaptive landscapes resulting from differences in ecological processes acting on different lineages.
On the other hand, for Anthropoids, their stability in $\mathbf{P}$-matrix structure may thus be possible as a consequence of stability of the functional relationships imposed over by the adaptive landscape in this lineage, as it seems that in this group, dietary shifts imposed changes on body size only, at least for Platyrrhines [@marroig_size_2010].

From an information theory point of view [@brooks_entropy_1989], natural selection will increase the correlation between information encoded in a population and the information that represents its environment; the recent suggestion that developmental systems share properties with machine learning algorithms [@watson_evolution_2013] only reinforces such view; furthermore, the regulation of developmental systems, through both genetic and epigenetic effects, may also be targeted by selection for both robustness and replicability [@hansen_epigenetics:_2011].
These different ways of thinking about developmental and functional interactions mean that the probabilistic distribution of phenotypes, as a consequence of either genetic information or epigenetic effects, will tend to match the properties of the distribution of fitness in a particular environment.
Thus, the maintenance of the same distribution over macroevolutionay timescales indicates that the fitness distribution itself may be stable as well.
