# Abstract

```{r options_ppca, include = FALSE}
opts_chunk $ set (fig.path = 'Figures/', dev = 'pdf')
```

```{r captions_ppca, include = FALSE}
captions $ fig.matrix_variation <- 'Representation of the steps used to characterize covariance matrix variation. In (a), the set of covariance matrices $\\mathbf{A}$, $\\mathbf{B}$ and $\\mathbf{C}$ in the neighbourhood of $\\mathbf{M}$ are projected into an Euclidean space and eigentensors are estimated (PM1 and PM2); in (b), these eigentensors are rotated to incorporate phylogenetic relatedness; in (c), covariance matrices corresponding to the upper and lower bounds of the confidence intervals for each axis are returned back to the original manifold. See text for more details. \\label{fig:matrix_variation}'

captions $ fig.eval <- 'Posterior distribution of eigenvalues obtained for pPCs; positive eigenvalues are associated with phylogenetic signal; negative eigenvalues are associated with convergence. \\label{fig:eval}'

captions $ fig.loess <- 'Distribution of mean SRD scores for each trait along the entire ordered set of phylogenetic PCs, separated into skull regions. \\label{fig:loess}'

captions $ fig.moran <- 'Posterior bivariate distribution of variances explained by each phylogenetic PC *versus* Moran\'s Indexes estimated for each axis. \\label{fig:moran}'

captions $ fig.phylo_full <- 'Phylogenetic decomposition of $\\mathbf{P}$-matrix variation. (a) Decomposition of matrix diversity over the phylogenetic hypothesis for Anthropoidea; the size of each circle indicates the percentage of diversity on each node, according to the legend. (b) Mean $\\mathbf{P}$-matrices of each species projected over the first three and the last phylogenetic Principal Components (G1-3 and L1, respectively); cell colors represent projection values, according to the legend. \\label{fig:phylo_full}'

captions $ fig.srd_gl <- 'Covariance structure variation associated with the first three and last  pPCs (Global 1-3 and Local 1, respectively), represented using posterior mean SRD scores. Dotted lines represent average SRD scores for each comparison. Traits are colored according to their assocation with each skull region, according to the legend. \\label{fig:srd_gl}'

captions $ fig.srd_shape <- 'Mean SRD scores for the first three and the last phylogenetic PCs (Global 1-3 and Local 1, respectively) represented over the mean skull shape of diana guenons (*Cercopithecus diana*). \\label{fig:srd_shape}'

### if thesis

captions $ fig.rs_riem <- 'The relationship between Riemannian Distances and Random Skewers among pairs of $\\mathbf{P}$-matrices. Each observation is the posterior mean value for each statistic, estimated from $100$ posterior samples obtained for each OTU. Colors indicate the lower sample size in each pair, according to the legend. \\label{fig:rs_riem}'

captions $ fig.ss_local1 <- 'Relationship between sample sizes and the first local phylogenetic Principal Component. \\label{fig:ss_local1}'
    
```

# Introduction

Phenotypic traits evolve in a concerted manner either because of shared genetic and developmental processes or joint effects on fitness [@gould_spandrels_1979; @lande_quantitative_1979; @felsenstein_phylogenies_1988; @zeng_long-term_1988], and response to natural selection is maximized when both effects coincide, while discordance between them may hinder a direct response  [@schluter_adaptive_1996].
The additive genetic covariances among traits ($\mathbf{G}$) and the partial regression coefficients of fitness onto traits ($\beta$) provide linear approximations for both these effects in the characterization of phenotypic change across generations [@lande_quantitative_1979; @rice_general_2002].
Therefore, the additive covariances in $\mathbf{G}$ represent the codependency among traits due to pleiotropy and linkage disequilibrium, characterizing a linear genotype/phenotype map centered on the mean phenotype [@cheverud_developmental_1996; @wagner_homologues_1996; @wagner_eigenvalue_1984].

However, this structure of pleiotropic interactions depends on the local curvature of the genotype/phenotype map, traditionally represented in quantitative genetics as either dominance or epistasis [@rice_evolution_1998; @wolf_developmental_2001; @rice_developmental_2004].
The effect of epistatic *loci* on the modulation of pleiotropic interactions has been identified in experimental settings [e.g. @cheverud_pleiotropic_2004; @wolf_epistatic_2005; @pavlicev_genetic_2008], thus indicating that populations may harbor genetic variation in the linear components of genotype/phenotype maps.
Genetic covariances among phenotypic traits then evolve as a consequence of shifts in allele frequencies in these *loci*, for example in response to genetic drift and founder effects [@goodnight_quantitative_2000; @de_brito_evolution_2005].
Local features of the adaptive landscape may also impact genetic covariances among traits, as $\mathbf{G}$ is thought to match the patterns imposed by stabilizing selection and mutational effects [@lande_genetic_1980; @cheverud_quantitative_1984; @jones_mutation_2007]. Moreover, more recently, both experimental data [@careau_evolution_2015] and simulation-based models [@jones_effects_2012; @melo_directional_2015] have demonstrated the effects of directional selection on the structure described by $\mathbf{G}$.

The relationship between adaptive landscapes and intrapopulational covariance structure is mediated through performance [@arnold_morphology_1983], which may be thought as a dynamical property of phenotypes; thus, the separation between developmental and functional interactions as two distinct factors shaping phenotypic covariance structure is blurry at least [@cheverud_developmental_1996; @zelditch_epigenetic_2011].
For example, in the mammalian skull, precursor tissues (paraxial mesoderm and neural crest) originate two distinct regions, Face and Neurocranium, which exhibit marked contrasts in terms of functions which they perform, interactions with soft tissues, and response to developmental milestones, such as birth or weaning [@hallgrimsson_mouse_2008; @lieberman_epigenetic_2011].
Such contrast is the result of distinct gene expression profiles, which are further changed by the diffusion of signaling factors, thus generating a feedback loop of cell and tissue differentiation [@turing_chemical_1952; @marcucio_molecular_2005; @meinhardt_models_2008; @franz-odendaal_epigenetics_2011; @xu_correlations_2015].

These signaling factors may be targeted at specific cell lineages, but the contact between adjoining tissues may produce correlated changes between them due to mechanical interactions and through mutual signaling cascades that induce specific behaviors in neighbor tissues, and such interactions are necessary for the proper development of both neighbors [@cheverud_effects_1992; @ravosa_masticatory_2000; @jiang_tissue_2002; @marcucio_mechanisms_2011].
For instance, cranial vault growth, which occurs on the final stages of pre-natal development, is a result of the tension exerted by the growing brain on its encircling membranes, inducing them to secrete growth factors which promote bone growth [@opperman_cranial_2000; @rice_molecular_2003]; furthermore, post-natal facial growth is induced by muscular activity related to masticatory function, and this effect dominates post-natal skull growth [@zelditch_ontogeny_1992; @herring_muscle-bone_2011]
Thus, development is composed of a series of such events, and it may be difficult to isolate the effect each individual process has on covariance structure, given their spatial overlap [@hallgrimsson_deciphering_2009]; hence, both the temporal hierarchy and the spatial organization of developmental and functional interactions influence the patterns embedded in genetic covariance structure.

Empirical evidence on long-term changes on genetic covariance structure [comparative quantitative genetics; @steppan_comparative_2002] rely on the correspondence between patterns embedded in $\mathbf{G}$ and its phenotypic counterpart $\mathbf{P}$, since estimating $\mathbf{G}$ demands high sample sizes and nonetheless such estimate is prone to substantial error, since sample units are families rather than individuals [@meyer_estimating_1991; @houle_estimating_2015].
The similarity in covariance structure between $\mathbf{P}$ and $\mathbf{G}$ ['Cheverud's Conjecture'; @cheverud_comparison_1988; @roff_estimation_1995], has been supported in different trait systems and organisms [e.g.: @waitt_genetic_1998; @dochtermann_testing_2011; @garcia_quantitative_2014], indicating that environmental effects ($\mathbf{E}$) are either uncorrelated or exhibit patterns similar to $\mathbf{G}$, since these effects exert their influence over phenotypes through the same developmental processes in which genetic variation is structured [@rice_general_2002; @rice_developmental_2004].

The comparative analysis of phenotypic covariance structure shows that $\mathbf{P}$ can remain stable in macroevolutionary scales [e.g. @marroig_comparison_2001; @oliveira_covariance_2009; @kolbe_convergent_2011], and such stability may be a consequence of alignment between phenotypic covariance structure and the local features of the adaptive landscape acting over different lineages.
In New World Monkeys (Platyrrhini), divergence in body size among lineages is closely associated with shifts in diet composition [@rosenberger_evolution_1992].
Size represents the main feature of both genetic [@cheverud_morphological_1995; @cheverud_quantitative_1996] and phenotypic [@marroig_size_2005] covariance structure, and for some groups within New World Monkeys, such as Atelids and Callithrichines, divergence in body size is a direct consequence of directional selection [@marroig_size_2010; @marroig_modularity_2012].
However, dietary or locomotory divergence between sister lineages can rearrange the patterns expressed in $\mathbf{P}$ in some situations [@young_serial_2005; @young_development_2010; @monteiro_adaptive_2010; @haber_evolution_2015], and such reorganization hints at changes in the underlying architecture of the genotype/phenotype map, indicating that the patterns embedded in the adaptive landscape can overcome constraints imposed by development [@jamniczky_comparison_2009].
The adaptive radiation of phylostomid bats, for instance, has involved rearrangements of mandibular phenotypic covariance structure [@monteiro_adaptive_2010], as the dietary divergence in this group is probably associated with a very heterogeneous adpative landscape.

Currently, there is an abundance of methods dedicated to the comparison of covariance structure among lineages. 
These methods will either provide a sole descriptive statistic of matrix similarity or dissimilarity [@krzanowsky_between-groups_1979; @mitteroecker_ontogenetic_2009; @bookstein_comparing_2014] or rely on some form of hypothesis testing [@phillips_hierarchical_1999; @cheverud_comparing_2007].

### Objectives

The interplay between developmental and functional interactions, and their relationships with the structure of adaptive landscapes may both influence the divergence between covariance structure in sister lineages. Thus, in the present work, we evaluate the stability of cranial phenotypic covariance structure along the diversification of Anthropoid Primates under an explicit phylogenetic framework, considering that either developmental and functional interactions will produce different degrees of stability among different cranial regions, thus producing a non-random pattern of changes in covariance structure.

# Methods

## Sample

Our sample consists of `r sum (Aux $ sample.size)` individuals, distributes across `r length (Aux $ sample.size)` species. 
These species are distributed throughout all major Anthropoid clades above the genus level, comprising all Platyrrhini genera and a substantial portion of Catarrhini genera.
We associate this database with a ultrametric phylogenetic hypothesis for Anthropoidea (\autoref{fig:phylo_model}), derived from @springer_macroevolutionary_2012.

Individuals in our sample are represented by 36 registered landmarks, using either a Polhemus 3Draw or a Microscribe 3DS for Platyrrhini and Catarrhini, respectively.
Twenty-two unique landmarks represent each individual (Figure \ref{fig:landmarks}), since fourteen of the 36 registered landmarks are bilaterally symmetrical. For more details on landmark registration, see @marroig_comparison_2001 and @oliveira_covariance_2009.
Databases from both previous studies were merged into a single database, retaining only those individuals in which all landmarks from both sides were present.
In the present work, we considered only covariance structure for the symmetrical component of variation; therefore, prior to any analysis, we controlled the effects of variation in assymmetry.
We followed the procedure outlined in @klingenberg_shape_2002 for bilateral structures by obtaining for each individual a symmetrical landmark configuration, averaging each actual shape with its reflection along the sagittal plane.

We used this database to obtain local shape variables [@marquez_measurement_2012], which represent infinitesimal volumetric expansions or retractions, calculated as the natural logarithm determinants of derivatives of the TPS function between each individual in our sample and a reference shape (in our case, the mean shape for the entire sample, estimated from a Generalized Procrustes algorithm).
Such derivatives were evaluated at the midpoints between pairs of landmarks represented in \autoref{fig:landmarks}, for a total of 38 local shape variables.

After obtaining these values, we estimated covariance $\mathbf{P}$-matrices for size (represented by the natural logarithm of Centroid Size) and local shape variables after removing fixed effects of little interest in the present context, such as sexual dimorphism, for example.
These effects were removed through a multivariate linear model adjusted for each species, according to \autoref{fig:phylo_model}.
We adjusted such models under a Bayesian framework, sampling $100$ residual covariance matrices from the posterior distribution of each model.
These distributions allow us to estimate uncertainty for any parameters derived from these matrices as credibility intervals; furthermore, since posterior distributions are conditional upon the prior distribution we used --- a uniform Wishart distribution --- every matrix sampled from these posterior distributions is also a realization of a Wishart distribution, therefore positive-definite regardless of sample size [@gelman_bayesian_2004]; in this framework, lower sample sizes imply in broader, less informative, credibility intervals.
For each posterior sample, we estimated geometric mean covariance matrices, since such mean respects the underlying structure of the Riemannian manifold in which positive-definite symmetric matrices lie [@moakher_differential_2005; @moakher_averaging_2006]; thus, these mean $\mathbf{P}$-matrices are also positive-definite, regardless of sample size.
For each species, we ran independent models, with 13000 iterations of MCMC sampling, discarding the 3000 initial iterations as a burnin period and further sampling one covariance matrix per 100 iterations to avoid autocorrelations induced by sequential sampling.

## Phylogenetic Decompostion of Matrix Diversity

In order to evaluate the distribution of covariance structure diversity during Anthropoid diversification, we estimated Riemannian distances among all pairs of mean $\mathbf{P}$-matrices, according to the definition given by @mitteroecker_ontogenetic_2009; for any pair of positive-definite covariance matrices $\mathbf{C}_i$ and $\mathbf{C}_j$ of size $p \times p$, the distance $d(\mathbf{C}_i, \mathbf{C}_j)$ is given by
\begin{equation}
d(\mathbf{C}_i, \mathbf{C}_j) = \sqrt{\sum_{k = 1}^p \ln^2 \lambda_k(\mathbf{C}_i\mathbf{C}_j^{-1})}
\label{eq:riemdist}
\end{equation}
where $\lambda_k(\cdot)$ refers to the $k$-th eigenvalue obtained from the spectral decomposition of a given matrix, in this case the product $\mathbf{C}_i\mathbf{C}_j^{-1}$.
This distance among pairs of $\mathbf{P}$-matrices is negatively correlated with Random Skewers comparisons (\autoref{fig:rs_riem}), a measurement of matrix similarity explored elsewhere [@cheverud_comparing_2007]; such correlation indicates that our exploration of Riemmanian distances among $\mathbf{P}$-matrices could be extended to other types of measurement of similarity or dissimilarity among these objects without hindrances. 

```{r rs_riem, echo = FALSE, fig.width = 8, fig.height = 8, fig.cap = captions $ fig.rs_riem}
if (thesis) ppca.extra $ rs.riem
```

Using these distances among $\mathbf{P}$-matrices, we estimate matrix diversity at each node of the phylogenetic tree of Anthropoidea using a measurement of the weighted distance among the distributions of matrix distances for the two descending edges, based on @pavoine_decomposition_2010.
For a fully resolved tree, diversity $w_i$ on node $i$ is given by
\begin{equation}
w_i = \frac{1}{2} \frac{n_\alpha n_\beta} {n_i n_T} D_{\Delta}^2(P_\alpha, P_\beta)
\label{eq:div}
\end{equation}
where $\alpha$ and $\beta$ represent the subsets of descendants from node $i$, and $n$ refers to the number of OTUs on each set ($n_i$ for the total descendants of node $i$; $n_T$ for the total number of OTUs considered; $n_\alpha$ and $n_\beta$ for the size of descending subsets).
$D_{\Delta}(P_\alpha, P_\beta)$ represents the actual distance between the two  distributions $P_\alpha$ and $P_\beta$ for descending nodes, as formulated by @rao_diversity_1982:
\begin{equation}
D_{\Delta} (P_\alpha, P_\beta) =
\sqrt{2 \bigg( 2 H_{\Delta} \bigg( \frac{P_\alpha + P_\beta}{2} \bigg) -
H_{\Delta}(P_\alpha) - H_{\Delta}(P_\beta) \bigg)}
\label{eq:distdist}
\end{equation}
where
\begin{equation}
H_{\Delta} (P) = \sum_{i,j \in P} \frac{d^2(\mathbf{C}_i, \mathbf{C}_j)}{2}
\label{eq:rao}
\end{equation}
represents Rao's quadratic entropy among Riemannian distances $d(\mathbf{C}_i, \mathbf{C}_j)$ as defined in \autoref{eq:riemdist}.

Following the framework estabilished by @pavoine_decomposition_2010, diversity $w_i$ can be normalized as $v_i = w_i / \sum_i w_i$ to represent the percentage of diversity with respect to the total diversity on the phylogenetic tree.
We test three different hypothesis regarding the distribution of $v_i$ values through Anthropoid diversification: (1) that $\mathbf{P}$-matrix diversity is concentrated in a single node; (2) that $\mathbf{P}$-matrix diversity is concentrated in a reduced number of nodes; (3) that $\mathbf{P}$-matrix diversity is skewed towards either the root or tips of the phylogeny, in a two-tailed test.
We test each hypothesis against the null hypothesis that the distribution of matrix diversity is randomly arranged over the phylogeny; such null hypothesis is represented by randomizing the association between terminal branches and covariance matrices, constructing $9999$ distributions of $v_i$ values that represent this scenario.
Each test is carried out using a different parameter derived from the distribution of $v_i$ values [described in detail by @pavoine_decomposition_2010], comparing the actual value obtained with a null distribution constructed using permutations.
The third hypothesis can be tested either by considering only the topology of the tree and by also considering branch lengths; both tests are similar to Blomberg's [-@blomberg_testing_2003] $K$ test, as they search for a phylogenetic signal in covariance structure diversity.

## Characterizing Covariance Matrix Variation

The tests described in the previous section allow us to pinpoint which nodes contribute mostly to divergence in covariance structure; however, these tests are not designed to properly describe the actual changes in $\mathbf{P}$-matrix structure that are responsible for such divergence. To actually represent these changes in a comprehensible manner, we combine a number of ordination techniques to reduce the dimensionality of the manifold that contains covariance matrices of size $p \times p$ (\autoref{fig:matrix_variation}).

For a Riemannian manifold, there exists at least one bijective function defined in the neighbourhood of a given covariance matrix $\mathbf{M}$ that maps the manifold to an Euclidean space --- a hyperplane with $p (p - 1) / 2$ dimensions also contained in $\mathbb{R}^{p \times p}$ --- and equips the manifold with a notion of inner product, thus allowing the construction of an orthonormal basis that can be used to describe variation in $\mathbf{P}$-matrix structure.
For a covariance matrix $\mathbf{X}$ in the neighbourhood of $\mathbf{M}$,
\begin{equation}
f(\mathbf{X}) = \log (\mathbf{M}^{- \frac{1}{2}} \mathbf{X} \mathbf{M}^{- \frac{1}{2}})
\label{eq:map}
\end{equation}
represents one possible function.
Here, the logarithm operator refers to matrix logarithm; for symmetric positive-definite matrices, this transformation is equivalent to applying the usual logarithm function to the eigenvalues of such matrix and reverting the spectral decomposition.
The function defined in \autoref{eq:map} also transforms the Riemannian distance among covariance matrices defined in \autoref{eq:riemdist} into Euclidean distances between transformed matrices [@moakher_differential_2005].

![`r captions $ fig.matrix_variation`](Figures/matrix_variation.pdf)

We defined the average matrix among all sampled $\mathbf{P}$-matrices as the location parameter $\mathbf{M}$ to map the entire set of posterior $\mathbf{P}$-matrices into an Euclidean space.
We then used these $\mathbf{P}$-matrices to produce axes of matrix variation using an eigentensor decomposition [@basser_spectral_2007; @hine_characterizing_2009] obtaining a set of eigentensors and eigenvalues that summarise matrix variation (\autoref{fig:matrix_variation}a).
As a consequence of using the mean covariance matrix for the entire sample over \autoref{eq:map}, the projections over eigentensors we obtained are naturally centered on $\mathbf{M}$. 

We used the projections of $\mathbf{P}$-matrices over these eigentensors as traits in a phylogenetic Principal Component Analysis [pPCA; @jombart_putting_2010], which produces a new set of axes of matrix variation that considers both trait dispersal and phylogenetic relationships among OTUs simultaneously. If $\mathbf{Z}$ represents a matrix with projections of $\mathbf{P}$-matrices over each eigentensor on its columns, phylogenetic PCs are the eigenvectors obtained from
\begin{equation}
\frac{1}{2n} \mathbf{Z}^t(\mathbf{W} + \mathbf{W}^t) \mathbf{Z}
\end{equation}
where $\mathbf{W}$ represents the matrix of phylogenetic distances between OTUs; here, the distance $w_{ij}$ between tips $i$ and $j$ is the sum of branch lengths from their last common ancestor to both tips. Other definition of phylogenetic distances may be used [see @jombart_putting_2010]; the results we show here were not changed by considering different measures of phylogenetic distance, regardless of whether these distances consider branch lengths among OTUs or not.

Such analysis produces both positive and negative eigenvalues, which are respectively associated with variation close to the root of the tree ('Global') and variation close to the tips ('Local') in matrix structure (pPC1 and pPC2 in \autoref{fig:matrix_variation}b, respectively). @pavoine_decomposition_2010 argues that this contrast between Global and Local components in phylogenetic PCs reflects phylogenetic signal and convergence in trait values, respectively, as observed in the distribution of Moran's [-@moran_interpretation_1948; -@moran_notes_1950] autocorrelation Indexes for each axis constructed in this manner. This index can be understood as the degree onto which an observed value in a given OTU is determined by the values on its phylogenetic "neighborhood" (as expressed by $\mathbf{W}$), in a similar manner to autoregressive models [@cheverud_autocorrelation_1985; @cheverud_quantitative_1985].

For each pPC obtained in this manner, we obtained two covariance matrices by estimating the upper and lower limits of the 95\% confidence interval for each axis and mapping these values back to the manifold of symmetric positive-definite matrices (\autoref{fig:matrix_variation}c), defining the inverse operation associated with \autoref{eq:map} as
\begin{equation}
f^{-1}(\mathbf{X}) = \mathbf{M}^{\frac{1}{2}}\exp(\mathbf{X})\mathbf{M}^{\frac{1}{2}}
\label{eq:inv}
\end{equation}
where the exponential operator refers to matrix exponential. We used these covariance matrices to describe matrix variation associated with each axis comparing each pair of matrices with the Selection Response Decomposition tool [@marroig_selection_2011], in order to pinpoint which traits are associated with the divergence in covariance structure associated with each pPC.

In order to characterize such divergence in covariance structure with respect to the uncertainty in $\mathbf{P}$-matrix estimation, we carried out the analyses described in this section with both mean $\mathbf{P}$-matrices obtained from posterior samples, and with posterior samples themselves, obtaining $100$ sets of phylogenetic PCs and $100$ sets of paired covariance matrices for each axis, thus allowing us to estimate a posterior distribution of mean SRD scores for each trait in all pPCs. We use the phylogenetic PCA estimated over mean $\mathbf{P}$-matrices in order to represent the phylogenetic patterns described by each pPC.

We use the posterior distribution of mean SRD scores over traits and pPC to investigate whether these changes in trait-specific covariance structure along Anthropoid diversification are randomly distributed with respect to the skull regions delimited in \autoref{tab:dist} by comparing SRD scores estimated within each region for all pPCs.

### Software

We performed all analyses under R `r paste (version $ major, version $ minor, sep = '.')` [@r_core_team_r:_2015].
We fitted Bayesian linear models for estimating posterior $\mathbf{P}$-matrix samples using the *MCMCglmm* package [@hadfield_mcmc_2010].
Source code for the eigentensor decomposition can be found at <http://github.com/wgar84>.
The phylogenetic decomposition of diversity was provided by @pavoine_decomposition_2010 in their Supplemental Material, while pPCA is implemented in the *adephylo* package [@jombart_adephylo:_2010]; the SRD method is provided by the *evolqg* package [@melo_evolqg_2015].
In order to obtain symmetrical landmarks configurations, we used code provided by Annat Haber, available at <http://life.bio.sunysb.edu/morph/soft-R.html>. 

# Results

The distribution of covariance matrix diversity along the Anthropoid phylogeny (\autoref{fig:phylo_full}a) indicates that the divergence between Catarrhini and Platyrrhini contributes to approximately 10\% of all covariance matrix diversity; within New World Monkeys, the divergence between Atelidae and Cebidae contributes with 4\% to covariance structure diversity, while for Catarrhines, the divergence between Hominoidea and Cercopithecoidea contributes 3\% to overall covariance matrix diversity. The remaining $\mathbf{P}$-matrix diversity is distributed along the tree, with a consistent decay of explained diversity the closer any given node is from terminal branches.
With respect to the hierarchy of tests exploring the phylogenetic distribution of matrix diversity (\autoref{tab:riem_decdiv}), all tests reject their null hypotheses of random arrangements of diversity along the Anthropoid phylogeny, thus indicating that covariance matrix diversity exhibits some degree of phylogenetic structure, with these few basal nodes contributing to a greater extent to such diversity.

\input{Tables/riem_decdiv} <!-- tab:decdiv -->

```{r phylo_full, echo = FALSE, fig.width = 10, fig.height = 12, fig.cap = captions $ fig.phylo_full, warning = FALSE}
Tree.Plots $ ppca
```

The eigenvalue distribution of phylogenetic Principal Components that describe $\mathbf{P}$-matrix structure (\autoref{fig:eval}) shows that individual Global components surpass their Global counterparts in terms of explained variance, such that the first Global component has a larger contribution than the first Local component to interspecific $\mathbf{P}$-matrix variation. While there are less positive than negative eigenvalues, Global components explain more than half (57\%) of the total $\mathbf{P}$-matrix variation, considering mean absolute eigenvalues obtained from their posterior distribution. The posterior distribution of Moran's Index for phylogenetic Principal Components (\autoref{fig:moran}) is assymetric towards positive values, also indicating that the similarity produced by phylogenetic conservatism is greater than the similarity produced by convergence in $\mathbf{P}$-matrix structure.

```{r eval, fig.width = 6, fig.height = 6, echo = FALSE, warning = FALSE, message = FALSE, fig.cap = captions $ fig.eval}
post.ppca $ plot.ppca.eval + xlab ('Phylogenetic Principal Component') +
  scale_x_continuous(breaks = c(1, 31, 61, 91))
```

```{r moran, fig.width = 6, fig.height = 6, echo = FALSE, warning = FALSE, message = FALSE, fig.cap = captions $ fig.moran}
post.ppca $ plot.var.moran
```

Considering the distribution of matrix projections for each OTU on these pPCs (\autoref{fig:phylo_full}b), we observe that the first Global pPC separates New World and Old Monkeys, while the second Global pPC consists of a contrast between Atelids and Cebids, and the third Global pPC generally contrasts Hominids with the remaining Anthropoids, thus indicating a pattern of $\mathbf{P}$-matrix variation consistent with those observed in the diversity decomposition summarized in \autoref{fig:phylo_full}a. The first Local pPC consists of localized contrasts between sister species, such as the two representatives of *Saimiri*, for instance; however, matrix projections over this axis can be explained by the effect of sample sizes (\autoref{fig:ss_local1}). These localized contrasts can thus be explained on the account of substantial sample size differences between sister species.

```{r srd_gl, echo = FALSE, fig.width = 10, fig.height = 10, warning = FALSE, message = FALSE, fig.cap = captions $ fig.srd_gl}
post.ppca $ plot.srd.gl12 +
  scale_color_brewer(name = 'Region',
                     labels = c('Size', 'Oral', 'Oral/Nasal', 'Nasal', 'Zygomatic',
                       'Orbit', 'Base', 'Vault'),
                     palette = 'Dark2')
```

```{r ss_local1, eval = thesis, echo = FALSE, fig.width = 8, fig.height = 8, fig.cap = captions $ fig.ss_local1}
ppca.extra $ ss.local1
```

The variation in trait-specific covariance structure described by these four phylogenetic Principal Components, as captured by comparing the posterior distribution of confidence intervals for each axis using Selection Response Decomposition (\autoref{fig:srd_gl}) indicates that, regardless of which axis is considered, traits with lower posterior SRD scores are usually localized in either Orbit or Basicranium, along with log Centroid Size, which represents the covariance structure associated with allometric relationships.
Traits in remaining skull regions (Oral, Nasal, Zygomatic, Vault) consistently exhibit higher SRD scores, thus indicating a more stable covariance structure associated with these regions throughout Anthropoid diversification. 

```{r loess, fig.width = 10, fig.height = 10, echo = FALSE, warning = FALSE, message = FALSE, fig.cap = captions $ fig.loess}
alt.spline
```

The overall distribution of average posterior SRD scores along the entire set of phylogenetic PCs (\autoref{fig:loess}) indicates that this behavior detailed on \autoref{fig:srd_gl} for G1-3 and L1 is the norm also for the remaining pPCs, that is, Orbit and Basicranial traits along with allometric relationships consistenly have lower average SRD scores than other skull regions. Notice that the explained amount of variance for intermediate pPCs (roughly from pPC 31 to 61) is very close to zero (\autoref{fig:eval}); thus, although matrices describing confidence intervals in these axes can be recovered, the pattern expressed by then should not be taken into account. 

# Discussion

1. Adaptive landscapes and ecological processes

2. Development as trajectories through phenotype space
